packages/image_picker/image_picker_android/example/lib/main.dart

```diff
@@ -19,6 +20,11 @@ void appMain() {
 }
 
 void main() {
+  // Set to use Android Photo Picker.
+  // #docregion photo-picker-example
+  final ImagePickerPlatform platform = ImagePickerPlatform.instance;
+  (platform as ImagePickerAndroid).useAndroidPhotoPicker = true;
```

Comment:
This code will crash on other platforms, which is fine in the context of the implementation package's example, but terrible for the README. Let's use:

```
if (platform is ImagePickerAndroid) {
  platform.useAndroidPhotoPicker = true;
}
```

Also, I think `imagePickerImplementation` would be a better name. "implementation" is a better description in this context of what it is, and giving it a more specific name makes it easier for the reader to drop this code into their app. E.g., imagine a reader who is copying this code from here, and similar platform-specific code from one of our other plugins, and what happens if both just use "platform" or "implementation".